samples/nrf9160/lwm2m_client/src/ui/ui_buzzer.c

```diff
@@ -59,12 +57,12 @@ int ui_buzzer_on_off(bool new_state)
 
 	if (frequency == 0) {
 		/* Turn off buzzer when frequency = 0 */
-		ret = pwm_pin_set_usec(buzzer_dev, buzzer_pin, USEC_PER_SEC, 0, 0);
+		ret = pwm_set_pulse_dt(&buzzer, 0);
 	} else {
 		uint32_t period = PERIOD(frequency);
 		uint32_t pulse_width = calculate_pulse_width(period, intensity);
 
-		ret = pwm_pin_set_usec(buzzer_dev, buzzer_pin, period, pulse_width * state, 0);
+		ret = pwm_set_dt(&buzzer, PWM_USEC(period), pulse_width * state);
```

Comment:
If `period` is USEC, should the last argument be also wrapped with `PWM_USEC`?

Reply:
Yes, fixed.